plenum/server/batch_handlers/batch_request_handler.py

```diff
@@ -24,11 +24,11 @@ def commit_batch(self, txn_count, state_root, txn_root, pp_time, prev_result):
                             txn_root)
 
     @abstractmethod
-    def post_batch_applied(self, state_root):
+    def post_batch_applied(self, ledger_id, state_root, pp_time, prev_result=None):
```

Comment:
When we will implement pluggable request handlers, we could include audit_batch_handler as a handler for every batch in write_request manager. That will allow us to avoid ledger_id param in every batch_handler

Reply:
While we did not implement pluggable rh, we probably can implement audit_batch_handler in an old style request handlers